pandas/core/series.py

```diff
@@ -4564,6 +4565,96 @@ def replace(
             method=method,
         )
 
+    @Substitution(
```

Comment:
I don't think the Substitution decorator should be necessary with the doc decorator. (and not seen them used together)

The doc decorator was created to supersede the Appender and Substitution decorators.

Reply:
I guess that Substitution is still necessary if we use one generic docstring for DataFrame and Series info. I could not figure out how I can replace some keywords in the base docstring, to make it suitable for both frame and series.